### ISSUE TITLE
master does not start (file missing)

### ISSUE THREAD

**Issue by sylvain-bougnoux** [spyder-ide/spyder#7722]
Today's version crashes on init

```
Traceback (most recent call last):
  File "D:\Bougnoux\Contrib\spyder\spyder\app\mainwindow.py", line 3211, in main
    mainwindow = run_spyder(app, options, args)
  File "D:\Bougnoux\Contrib\spyder\spyder\app\mainwindow.py", line 3075, in run_spyder
    main.setup()
  File "D:\Bougnoux\Contrib\spyder\spyder\app\mainwindow.py", line 839, in setup
    self.lspmanager.start_lsp_client('python')
  File "D:\Bougnoux\Contrib\spyder\spyder\plugins\lspmanager.py", line 712, in start_lsp_client
    language_client['instance'].start()
  File "D:\Bougnoux\Contrib\spyder\spyder\utils\code_analysis\lsp_client.py", line 133, in start
    creationflags=creation_flags)
  File "c:\program files\python36\lib\subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "c:\program files\python36\lib\subprocess.py", line 997, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] file does not exist
```
Sorry I tried to debug which file but I could not.
Back to last week commit works (eg b34a22a7ffafc2f661643fb2bfd5fb4926e5f24b).

BTW the report issue window popped up but disappear instantaneously. I don't know why either. Maybe it is a separate issue?

**Comment by jnsebgosselin** [comment]
I think PR #4751 added new dependencies that need to be installed in order for Spyder to run.

Those are `coloredlogs`,  `python-language-server[all]`,  `pydocstyle` and `pexpect`

After installing those, Spyder is starting correctly.

@ccordoba12 I do not see these new dependencies in `requirements.txt` or `test_requirements.txt`, should they be added there?

**Comment by jnsebgosselin** [comment]
... but then `Go to definition` and `Help` is broken for me while working perfectly fine before this. So I don't know if its the right thing to do or if there is something else to install.

**Comment by ccordoba12** [comment]
We're in the process of adding a new code completion architecture. Things are going to be a bit rough during the next couple of days, so we recommend to stay with beta1 until beta2 is released.

By the way, @jnsebgosselin was right about the new packages that need to be installed.

**Comment by jnsebgosselin** [comment]
@ccordoba12 Thanks for your answer! Help is working after all, its the go to definition that is not. 

I think I was able to pinpoint why it is not working. Should I open a new issue for this or you guys got it all covered already?

**Comment by jnsebgosselin** [comment]
Oh nice!

I've explained what I think the reason of the problem is in Issue #7726. I've created the issue already since I do not think the the split plugin merge will have an impact on this and it was already ready.

**Comment by ccordoba12** [comment]
It's not that that will interfere, it's just that all new work needs to be based on the layout defined by that PR.

**Comment by ccordoba12** [comment]
Closing because this should be fixed now by #7822.

**Comment by sylvain-bougnoux** [comment]
Yes it works for me (on windows). Thanks.
 
De : Carlos Cordoba <notifications@github.com> 
Envoyé : lundi 10 septembre 2018 19:34
À : spyder-ide/spyder <spyder@noreply.github.com>
Cc : sylvain-bougnoux <bougnoux@imra-europe.com>; Author <author@noreply.github.com>
Objet : Re: [spyder-ide/spyder] master does not start (file missing) (#7722)
 
Closing because this should be fixed now by #7822.

**Comment by ccordoba12** [comment]
Great!! Thanks for letting us know about it.
